constants.go

```diff
@@ -43,9 +45,9 @@ const (
     <graphics type='vnc' autoport='yes' listen='127.0.0.1'>
       <listen type='address' address='127.0.0.1'/>
     </graphics>
-    <interface type='network'>
+    <interface type='bridge'>
```

Comment:
so we don't need to create the `crc` network during setup an ymore?

Reply:
Yes we do, I'm only changing how it's used here. The `crc` network lives on system connection and we run the VMs on session connection after this change. With `network` type interface, libvirt will assume a network by that name on the same connection. With `bridge`, it'll access it as a bridge on the system, which `crc` network creates.

Reply:
Might be worth having that explanation in the commit log.